cli/src/main/java/org/jboss/as/cli/Util.java

```diff
@@ -266,21 +273,101 @@
 
     public static final String NOT_OPERATOR = "!";
 
-    private static final String ENCODING_EXCEPTION_MESSAGE = "Encoding exception.";
+    private static TerminalColor ERROR_COLOR;
+    private static TerminalColor SUCCESS_COLOR;
+    private static TerminalColor WARN_COLOR;
+    private static TerminalColor REQUIRED_COLOR;
+    private static TerminalColor BATCH_PROMPT_COLOR;
+    private static TerminalTextStyle BOLD_STYLE = new TerminalTextStyle(CharacterType.BOLD);
+    private static String ENCODING_EXCEPTION_MESSAGE = "Encoding exception.";
 
     private static Logger LOG = Logger.getLogger(Util.class);
 
+    public static final String formatErrorMessage(String message) {
+        return new TerminalString(message, ERROR_COLOR, BOLD_STYLE).toString();
+    }
+
+    public static final String formatSuccessMessage(String message) {
+        return new TerminalString(message, SUCCESS_COLOR).toString();
+    }
+
+    public static final String formatWarnMessage(String message) {
+        return new TerminalString(message, WARN_COLOR, BOLD_STYLE).toString();
+    }
+
+    public static TerminalString formatRequired(TerminalString name) {
+        return new TerminalString(name.toString(), REQUIRED_COLOR, BOLD_STYLE);
+    }
+
+    public static String formatBatchPrompt(String prompt) {
+        return new TerminalString(prompt, BATCH_PROMPT_COLOR).toString();
+    }
+
     public static boolean isWindows() {
         return WildFlySecurityManager.getPropertyPrivileged("os.name", null).toLowerCase(Locale.ENGLISH).indexOf("windows") >= 0;
     }
 
     public static boolean isSuccess(ModelNode operationResponse) {
-        if(operationResponse != null) {
+        if (operationResponse != null) {
             return operationResponse.hasDefined(Util.OUTCOME) && operationResponse.get(Util.OUTCOME).asString().equals(Util.SUCCESS);
         }
         return false;
     }
 
+    public static void formatPrompt(StringBuilder buffer) {
```

Comment:
I am wandering if a single color for both mode and host would not help visibility. User would have the ability to change the color.